ds701_book/13-Learning-From-Data.qmd

```diff
@@ -347,7 +387,7 @@ These suggest techniques we may use to avoid overfitting:
 
 2. Limit the complexity of the model.  Model complexity is often controlled via __hyperparameters__.
 
-### More Training Data
+## More Training Data
 
 It's not necessarily true that a order-3 polynomial is best for this problem.
 
```

Comment:
The line below about the Taylor series of sine is incorrect. The Taylor series of a Sine function includes all odd-order polynomial terms, i.e., $sin(x) = \sum_{n=0}^{\infty}(-1)^n \frac{x^{2n+1}}{(2n+1)!}$. I'd consider dropping that line since we can't really assume that students remember Taylor series.